llm/lorax/README.md

```diff
@@ -0,0 +1,102 @@
+# LoRAX: Multi-LoRA inference server that scales to 1000s of fine-tuned LLMs
+
+<p align="center">
+    <img src="https://github.com/predibase/lorax/blob/main/docs/images/lorax_guy.png" alt="LoRAX" style="width:200px;" />
+</p>
+
+[LoRAX](https://github.com/predibase/lorax) (LoRA eXchange) is a framework that allows users to serve thousands of fine-tuned LLMs on a single GPU, dramatically reducing the cost of serving without compromising on throughput or latency. It works by dynamically loading multiple fine-tuned "adapters" (LoRAs, etc.) on top of a single base model at runtime. Concurrent requests for different adapters can be processed together in a single batch, allowing LoRAX to maintain near linear throughput scaling as the number of adapters increases.
+
+## Setup
+
+First install SkyPilot and check that your cloud credentials are properly set:
+
+```shell
+pip install skypilot
+sky check
+```
+
+## Launch a deployment
+
+Create a YAML configuration file called `lorax.yaml`:
+
+```yaml
+resources:
+  cloud: aws
+  accelerators: A10G:1
```

Comment:
Maybe we can get rid of the `cloud`, and have multiple `accelerators` so that it can work for users that do not have AWS enabled:
```suggestion
  accelerators: {A10G, A10, L4, A100, A100-80GB}
```

Does that make sense?

Reply:
Done.